scale.yml

```diff
@@ -66,6 +66,7 @@
 - name: Upload control plane certs and retrieve encryption key
   hosts: kube-master | first
   environment: "{{ proxy_disable_env }}"
+  gather_facts: False
```

Comment:
`gather_facts: False` has been added to all other tasks in scale.yaml with https://github.com/kubernetes-sigs/kubespray/pull/6482
After the above PR, this task has been added without `gather_facts: False` with https://github.com/kubernetes-sigs/kubespray/pull/6822
This seems a valid change for me.

/lgtm

Reply:
what should I do for merge?

Reply:
@floryut Can you take a look at this PR?

Reply:
👍